isthisstockgood/templates/loading.html

```diff
@@ -90,7 +90,7 @@
 
             this.shadowRoot.append(style, wrapper);
 
-            this.animationTimeout = None;
```

Comment:
Appeared in the browser console as:

![cant-find-variable-none](https://github.com/user-attachments/assets/763a6c57-11cd-437d-a8a8-4bf93b5258b3)